scripts/commands.py

```diff
@@ -179,7 +205,6 @@ def browse_website(url, question):
 
     return result
 
-
```

Comment:
Rollback the whiteline corrections. We'll do them in a separate PR (which is already there, just not merged yet)